spec/plurimath/latex/parser_spec.rb

```diff
@@ -9501,7 +9501,17 @@
           Plurimath::Math::Symbol.new("*"),
           Plurimath::Math::Symbol.new("x"),
           Plurimath::Math::Unicode.new("&#x3e;"),
-          Plurimath::Math::Function::Text.new("\"tolerance\""),
+          Plurimath::Math::Symbol.new("\""),
```

Comment:
@suleman-uzair what does this change mean?

Reply:
While making changes to constants for `double prime` parsing, I realized that latex doesn't support double prime for text wrapping like AsciiMath so I updated related parser rules and specs.

Reply:
According to my testing in Latexmath gem and in latex editor `\text{…} `is not the same as `"..."` so that is why I changed this.
But if you want I can revert these changes. What do you suggest?

Reply:
![Screenshot from 2023-03-22 13-12-15](https://user-images.githubusercontent.com/96812483/226845679-3b3e8292-aefc-408b-a941-a701709d0db3.png)
![Screenshot from 2023-03-22 13-16-16](https://user-images.githubusercontent.com/96812483/226845688-e8e6cf04-7c33-430f-bd1f-2a6e2f8d6fea.png)
![Screenshot from 2023-03-22 13-16-18](https://user-images.githubusercontent.com/96812483/226845691-11c2b942-d642-4dcd-b68d-20a9949253b3.png)

Here in both screenshots of **AsciiMath**, we can see the `double quotes` convert math equations to simple text.
But in `Latex`'s editor, `double quotes` make no difference but the `\text` function keeps the spaces of the text and makes it readable.

Reply:
I am fine with this parsing, but the point is about the formula itself not being accurate. The "\text" portion should be specified in the formula. Can we add a formula to test the "\text{tolerance}" string, maybe in a new PR? Thanks.